.github/workflows/makefile.yml

```diff
@@ -28,6 +34,9 @@ jobs:
       with:
         ref: ${{ inputs.branch }}
     
+    - name: Commit SHA
+      run: git show --format="%h" --no-patch
+
```

Comment:
master should merged with changes from dev before testing

Reply:
May you explain more in detail that? You mean changes from dev as workflows folder?

Reply:
It looks  for me only master branch will be tested, however it should be dev branch with latest master changes.

Reply:
Yes that's the point. I will test against latest changes and if they are ok, the changes will be merged into dev branch. Master branch contains latest changes and dev branch is behind master

Reply:
Upstream (bminor/binutils) contains C1 <- C2 (Where C1, C2 are commits). Master and dev contain C1. After the first job, master will contain C2 as it was synced with upstream. After that a reusable workflow is called against the master branch. We are testing on C2, not on C1 as C1 already was tested and merged into dev. If testing and build is successful, C2 will be merged into dev branch.